### ISSUE TITLE
Get predicates from meta knowledge graph

### ISSUE THREAD

**Issue by karafecho** [RobokopU24/Issues#201]
This issue is to suggest that the UI supports more than a handful of Biolink predicates in the dropdown menu selection. While ROBOKOP will return more specific predicates in response to a more generic predicate, when available, I think it would be helpful for users to be able to choose upfront from a complete list of Biolink predicates that are available in the underlying KG.

**Comment by EvanDietzMorris** [comment]
The list of predicates that are actually in the graph can be pulled from the plater /metadata json, which is probably a better place to get them than biolink for the UI, but if we show them all at once it'd be quite a long list (64 currently).

**Comment by karafecho** [comment]
Agreed. I do NOT want to see Biolink predicates other than those that are actually available in the underlying KG. 

In terms of size, I don't think a list of 64 predicates will present a problem for users, especially if we can display them in some sort of structured form, rather than just a plain list. Users find it confusing to see predicates other than the one(s) they asked for in answer sets. If they see those predicates in the options, then they won't be as confused if they show up in the answer sets.

FWIW, the ARAX UI displays 65 predicates.

**Comment by cbizon** [comment]
I like the idea of using the metaknowledgegraph here over the biolink model.  It'll also solve the problem with treats not showing up #204

**Comment by Woozl** [comment]
Moved from #215 
> A related issue is that it's not always possible to restrict predicates to the desired level of specificity, given that the ROBOKOP question-builder is not supporting the full set of available predicates. For example, when I was attempting to create a more generic AOP sample query from the specific AOP sample query provided in #[278](https://github.com/RobokopU24/qgraph/issues/278), I had hoped to use more specific predicates than `related_to`, but the ROBOKOP question-builder tool doesn't support the desired predicates.
